docs/configuration/authentication/oidc_digid.rst

```diff
@@ -0,0 +1,68 @@
+.. _configuration_authentication_oidc_digid:
+
+======================================
+OpenID Connect voor inloggen met DigiD
+======================================
+
+Open Formulieren ondersteunt DigiD login voor burgers via het OpenID Connect protocol (OIDC).
+
+Burgers kunnen op die manier inloggen op Open Formulieren met hun DigiD account. In deze
+flow:
+
+1. Klikt een gebruiker in een formulier op de knop *Inloggen met DigiD via OpenID Connect*
+2. De gebruiker wordt via de omgeving van de OpenID Connect provider (bijv. Keycloak) naar DigiD geleid,
+   waar de gebruiker kan inloggen (met eventuele Multi Factor Authentication)
+3. DigiD stuurt de gebruiker terug naar de OIDC omgeving, die op zijn beurt de gebruiker weer terugstuurt naar Open Formulieren
+4. De gebruiker is ingelogd en kan verder met het invullen van het formulier
+
+.. _configuration_oidc_digid_appgroup:
+
+Configureren van OIDC voor DigiD
+================================
+
+Contacteer de IAM beheerders in je organisatie om een *Client* aan te
+maken in de omgeving van de OpenID Connect provider.
+
+Voor de **Redirect URI** vul je ``https://open-formulieren.gemeente.nl/digid-oidc/callback/`` in,
+waarbij je ``open-formulieren.gemeente.nl`` vervangt door het relevante domein.
+
+Aan het eind van dit proces moet je de volgende gegevens hebben (on premise):
```

Comment:
Waarom on-premise?

Reply:
Was gekopieerd van eerdere docs, maar is niet relevant denk ik, dus ik haal het weg